doc/developer-guides/hld/hld-virtio-devices.rst

```diff
@@ -216,9 +217,9 @@ virtqueues, feature mechanisms, configuration space, and buses.
 Virtio Framework Considerations
 ===============================
 
-How to realize the virtio framework is specific to a
+How to configure the virtio framework is specific to a
```

Comment:
"configure" doesn't seem the right word here.  The original "realize" meant "to cause something to happen", so if you don't like "realize" a better word here would be "implement".